packages/addons/driver/hdhomerun/changelog.txt

```diff
@@ -1,2 +1,5 @@
+7.0.101
+- version bump for LibreELEC 7.0 Beta 1
```

Comment:
Why not just say that you are changing the icon? beta 1 has nothing to do with it :)